mipengine/node/tasks/views.py

```diff
@@ -46,18 +48,50 @@ def create_view(
         A list of dataset names
     columns : List[str]
         A list of column names
-    filters_json : str(dict)
+    filters : dict
         A Jquery filters object
 
     Returns
     ------
     str
-        The name of the created view in lower case
+        The name of the created view
     """
     view_name = create_table_name(
         "view", command_id, context_id, node_config.identifier
     )
-    views.create_view(
+    views.create_pathology_view(
         view_name=view_name, pathology=pathology, datasets=datasets, columns=columns
     )
     return view_name.lower()
```

Comment:
why transform the table name you get from create_table_name to lower in so many places and not do it once inside create_table_name?

Reply:
Yeah good notice i will make it return lower in on create_table_name.